help/moderation.md

```diff
@@ -1,135 +1,45 @@
--   In some cases we may require that a paper be published in a
-    conventional journal before an appeal is considered.
-
-The appeals process allows moderators additional time to reconsider a
-submission, but repeated appeals with no additional information cannot
-be considered. If you send an appeal and the moderators reach the same
-decision as they did initially, no further appeal should be submitted.
-Extreme cases may be addressed to the appropriate [advisory
-committee](scientific_ad_board) chair only.
-
-Who are the arXiv moderators?
------------------------------
-
-[arXiv moderators](/moderators) are volunteer subject specialists who
-have been approved by their discipline-level [advisory
-committees](scientific_ad_board#advisory_committees) and by arXiv staff.
-
-Although the moderators may be publicly acknowledged, it is very
-inappropriate to contact any moderator directly regarding your
-submission. All communication about moderation decisions should be
-addressed to moderation@arxiv.org or, in extreme cases, the
-appropriate advisory committee chair. Moderators are encouraged not to
-reply to personal correspondence regarding arXiv submissions.
+# arXiv moderation
+
+arXiv is a forum for professional members of the scientific community. While arXiv prides itself on maintaining a permanent collection of scientific work that is made available to the public quickly and without cost, submissions to arXiv do require a minimal amount of moderation.
+
+## Why does arXiv moderate submissions?
+
+arXiv is an openly accessible, moderated repository for scholarly papers in specific scientific disciplines. Material submitted to arXiv is expected to be of interest, relevance, and value to those disciplines. arXiv reserves the right to reject or reclassify any submission.
+
+## What policies guide moderation?
+
+arXiv moderation may result in the *reclassification* or *removal* of a submission. 
+Reclassification by our moderators helps make sure that submissions are posted to the most appropriate archive. Cross-lists may be added to other related archives, or they may be removed by moderators when the classification is deemed inappropriate. A submission that is cross-listed to a category will still be announced in the mailings to that category.
+
+arXiv policies that could lead to the removal of a submission are:
+
+- *Unrefereeable content.* arXiv only accepts submissions in the form of an article that would be refereeable by a conventional publication venue. Papers that do not contain original or substantive research, including undergraduate research, course projects, and research proposals, news, or information about political causes (even those with potential special interest to the academic community) may be removed. Papers that contain inflammatory or fictitious content, papers that use highly dramatic and misrepresentative titles/abstracts/introductions, or papers in need of significant review and revision may be removed. 

```

Comment:
I'm flagging "undergraduate work" because, at least as written, identifies a class of people based on their student status who cannot have work accepted. I think the intention is that undergraduate theses may not be original or are incomplete, so for that reason, the work is unacceptable. I would suggest a modification to the text, like, "undergraduate course projects".